### ISSUE TITLE
FCIDUMP support

### ISSUE THREAD

**Issue by jeffhammond** [nwchemgit/nwchem#8]
The FCIDUMP format [1] seems to be rather popular for supporting implementations of exotic methods like FCIQMC.  As noted in the [HANDE manual](https://media.readthedocs.org/pdf/hande/v1.0/hande.pdf), Molpro, Q-Chem, and PSI4 all support this format.  PySCF also supports this format (http://sunqm.github.io/pyscf/tools.html).

I have written something along these lines before and will attempt to create a proper implementation unless someone else gets their first.  We do not document support for it that I can find, so if it is already supported, it is hidden from users.

[1] P. J. Knowles, N. C. Handy, Comp. Phys. Commun. 1989, 54, 75. *A determinant based full configuration interaction program.* https://doi.org/10.1016/0010-4655(89)90033-7

**Comment by dvorakm1** [comment]
Hi,
    Did you make any progress with the FCIDUMP for NWChem?  I would like to get all the one- and two-body integrals in the MO basis for my own external code.  I have been experimenting myself with the rimp2 module to get these but can't quite get it to work.
Thanks,
Marc

**Comment by jeffhammond** [comment]
No, but please look at https://github.com/nwchemgit/nwchem/issues/23#issuecomment-371276682, as you might be able to implement it in the TCE module yourself using the `eionly` method I noted.

**Comment by safischer** [comment]
Please, see [#23 (comment)](https://github.com/nwchemgit/nwchem/issues/23#issuecomment-387717386) for a method to dump the one and two electron integrals in the molecular orbital basis to a file.

**Comment by edoapra** [comment]
A preliminary implementation of FCIDUMP, following @safischer comment, is now available in the master branch.
Documentation at https://nwchemgit.github.io/FCIDUMP.html
Input example at https://github.com/nwchemgit/nwchem/blob/master/QA/tests/fcidump_hf/fcidump_hf.nw
